Doc/whatsnew/3.14.rst

```diff
@@ -92,6 +92,12 @@ ast
 Added :func:`ast.compare` for comparing two ASTs.
 (Contributed by Batuhan Taskaya and Jeremy Hylton in :issue:`15987`.)
 
+asyncio
+-------
+
+Made :class:`asyncio.Queue` an :term:`asynchronous iterable`.
+(Contributed by Wannes Boeykens in :gh:`120491`.)
```

Comment:
I'm assuming this is way to vague... suggestions?

Reply:
That actually looks great to me; it's not verbose but with the terms hyperlinked it's clear what this does.  

Optionally we could say something like ", which simplifies common producer-consumer patterns." - I'd match the style of the rest of the changelog.

Reply:
@nineteendo I agree with @Zac-HD that this is fine as the news item. 

I also really like how you went back and reframed the PR text. You went above and beyond my expectations, and I appreciate it :sunny: